motor/discovery/k8s/resolver.go

```diff
@@ -144,15 +144,17 @@ func (r *Resolver) Resolve(ctx context.Context, root *asset.Asset, tc *providers
 		return nil, err
 	}
 
+	if tc.IncludesDiscoveryTarget(common.DiscoveryAuto) {
+		log.Info().Msg("discovery option auto is used. This will detect the assets: cluster, cronjobs, pods, statefulsets, deployments, replicasets, daemonsets")
```

Comment:
We do not discover jobs. Was this intended in this PR?  https://github.com/mondoohq/cnquery/pull/176#issuecomment-1271474139

Reply:
That is a question for @imilchev

Reply:
yup! That seems to have slipped through. Please add it. Jobs should be discovered as well

Reply:
Added `jobs` to the auto mode discovery.